### DISCUSSION TITLE
How to solve  `SyntaxError: Cannot use import statement outside a module` isssuse

### DISCUSSION THREAD

**Discussion by Sachin-chaurasiya** [rehypejs/rehype#93] [Q&A] [answered]
I am using this package with react-markdown as `rehypePlugins`. When I was trying to build the project it was giving me this error.
```
src/main/resources/ui/node_modules/rehype-raw/index.js:6
    import {raw} from 'hast-util-raw'
    ^^^^^^

    SyntaxError: Cannot use import statement outside a module

```

**Comment by ChristianMurphy** [comment, marked as answer]
`react-markdown`, `rehype-raw`, and `hast-util-raw` are all ES Modules.
This works out of the box with Node and many build tools, a few build tools need to be configured, it looks like the build tool you are using may need to be configured.
[This ESM guide](https://gist.github.com/sindresorhus/a39789f98801d908bbc7ff3ecc99d99c) offers configuration suggestions for several build tools.

**Reply by Sachin-chaurasiya** [reply]
Thanks, @ChristianMurphy for the quick response.
